src/Resolvers/Microsoft.NET.Sdk.WorkloadManifestReader/Strings.resx

```diff
@@ -192,8 +192,8 @@
   <data name="SpecifiedManifestNotFound" xml:space="preserve">
     <value>Specified workload manifest was not found: {0}</value>
   </data>
-  <data name="SpecifiedWorkloadSetNotFound" xml:space="preserve">
-    <value>Specified workload set was not found: {0}</value>
+  <data name="WorkloadVersionFromGlobalJsonNotFound" xml:space="preserve">
+    <value>Workload version {0}, which was specified in {1}, was not found</value>
```

Comment:
Do we want users to run restore here, once restore is aware of the global.json changes? Is that why you haven't added that to the error yet? It would be best to give the user some action to take.

Reply:
I just hadn't thought of adding it when I was updating the string.